pipelines/respondent-home-ui.yml

```diff
@@ -167,6 +197,15 @@ jobs:
     file: ras-deploy/tasks/extract-source-from-release.yml
     input_mapping: { release-resource: respondent-home-ui-pre-release }
     output_mapping: { release-source: respondent-home-ui-pre-release-source }
+  - put: create-redis
+    resource: cf-cli-resource-preprod
+    params:
+      command: create-service
+      service: awselasticache-redis
+      plan: small
+      service_instance: ras-redis
```

Comment:
This should be `res-ops-redis` for preprod

Reply:
https://github.com/ONSdigital/respondent-home-ui/pull/89